data/quarg/quarg missions.txt

```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "station"
+	on offer
+		conversation
+			`As soon as you step off your ship a rather imposing Quarg blocks your exit ramp while stairing piercingly at your ship.`
+			`"It is as though a baby in swaddling has killed a dragon, and taken its treasure for its own..." it says with a mix of amusement and disgusted.`
+			choice
+				`	"It's a long story..."`
+					goto evasive
+				`	"I prefer to frame it as David and Goliath myself."`
+					goto righteous
+				`	"The Pug fancy themselves gods, but God is dead, soo..."`
+					goto anarchy
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	"It matters not how it happened."`
+				goto end
+			label righteous
+			`	"Did not your Jesus compare himself to this David also?" the Quarg says with what you assume is a sarcastic tone.`
```

Comment:
```suggestion
			`	"Did not your Jesus compare himself to this David also?" the Quarg says with what you assume is a sarcastic tone.`
```
This one hits me squarely wrong in many ways.

It demonstrates the Quarg have a specific understanding of a single religion that runs deeper than their grasp of the language as a whole, but also implies that they understand -only- such one specific religion; they are responding as one believer to another, not as an outsider. Even then, the phrasing here, more than the others, implies the player is a believer as well, which is entirely inappropriate.

This would be much better either as a response with a parable from a differing religion (the more different, the better) or simply outright ignorance of the reference.

Would be best served, imo, with replacing the initial reference entirely -in addition- to the above.

Reply:
I gotta agree with that, lets avoid references to actual earth religions.
Myths are fine though

Reply:
I don't know if I want the ES universe to have that religion cannon, especially in that state and with the same names, so yeah I'd prefer a more generic response, especially since not everyone will know about who this is and what they did if we use a specific story.
Also best not to open the can of worm of religious things, some may interpret it badly and whatnot, no need to risk it

Reply:
Would a reference to Hercules work? It’s from a religious body of work, just one that is no longer practice and has passed into myth.

Or should I aim for more generic than that?

Reply:
Updated this path to be more generic let me know if that works.